frontend/rolecall/src/app/services/response-status-handler.service.ts

```diff
@@ -65,6 +65,7 @@ export class ResponseStatusHandlerService {
   showError(errorEvent: ErrorEvent): Promise<string> {
     if (this.pendingErrors.has(errorEvent.url)) {
       // TODO: What should we return here?
+      // Is here a problem?
```

Comment:
The problem here is that we are returning an empty string, which is definitely not the intention of a function with `Promise<string>` type. The questions here are why is it returning a string in the first place, what are we doing with that string, and maybe we should change the return type to just void? It's not obvious to me what the return value means here.

But this fix might be out of scope of the current PR, so many just remove the added comment to get the rest of the fixes in?

Reply:
I thought the empty promise is like a promise no-op, or a promise when you know the returned value will be ignored. (Like the return value from a message dialog where OK is the only option.) This promise comes after displaying an error dialog and apparently the user has no option except to click OK.) I am happy to remove the TODO.

Reply:
I removed the extra comment but left the todo.